source/agora/test/Ledger.d

```diff
@@ -178,13 +178,17 @@ unittest
     auto txes = getChainedTransactions(gen_block.txs[$-1], period*10, gen_key_pair, 8);
     txes.each!(tx => node_1.putTransaction(tx));
 
+    bool blocks_equal;
     auto attempts = 80;  // wait up to 80*100 msecs (8 seconds)
     while (attempts--)
     {
-        if (containSameBlocks(nodes, 10))
-            return;
+        blocks_equal = containSameBlocks(nodes, 10);
+        if (blocks_equal)
+            break;
 
         // let them do catch-up after boot
         Thread.sleep(100.msecs);
     }
+
+    assert(blocks_equal, "Nodes do not contain the same blocks");
```

Comment:
This could just be `assert(0)` if you left the `return` instead of `break`

Reply:
Ping

Reply:
Alright.